app/ui/src/app/common/empty-state-card.component.ts

```diff
@@ -0,0 +1,30 @@
+import {
+  Component
+} from '@angular/core';
+
+@Component({
+  selector: 'syndesis-empty-state-card',
+  template: `
+  <div class="card empty-state-card">
+    <div class="card-pf card-pf-view card-pf-view-select card-pf-view-single-select">
+      <div class="card-pf-body">
+        <div class="card-pf-top-element">
+          <ng-content select=".empty-state-card__header"></ng-content>
+        </div>
+        <div class="card-pf-info text-center">
+          <ng-content select=".empty-state-card__body"></ng-content>
+        </div>
+      </div>
+    </div>
+  </div>
+  `,
+  styles: [`
+    :host { display: block; }
+    :host .card-pf, :host .card-pf:hover { box-shadow: none; }
+    :host .card-pf { background: transparent; padding-top: 1px; padding-bottom: 1px; }
+    :host .card-pf-view { border: 1px dashed #919191; }
+  `]
+})
+
```

Comment:
The overall shape of this CSS code turn it inot a good candidate to be promoted into its own SCSS file, linked to this component. That would allow you to use the Patternfly color variable fro `#919191` instead of that hardcoded reference in the last CSS statement.

Reply:
Can I just add the SCSS to `/app/common` adjacent to `empty-state-card.component.ts` or should I create a dir like `/app/common/empty-state-card/` and move the TS and SCSS there?

Reply:
As a rule of thumb, the convention is to create the SASS file right next to the component file, sharing its filename. In regards of where to save those files, perhaps moving those two it its own folder will help declutter an already cluttered folder as `common` ;)